x-pack/metricbeat/module/googlecloud/constants.go

```diff
@@ -73,3 +71,53 @@ const (
 	LabelUser     = "user"
 	LabelMetadata = "metadata"
 )
+
+// Available perSeriesAligner map
+// https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.alertPolicies#Aligner
+var AlignersMapToGCP = map[string]monitoringpb.Aggregation_Aligner{
+	"ALIGN_NONE":           monitoringpb.Aggregation_ALIGN_NONE,
+	"ALIGN_DELTA":          monitoringpb.Aggregation_ALIGN_DELTA,
+	"ALIGN_RATE":           monitoringpb.Aggregation_ALIGN_RATE,
+	"ALIGN_INTERPOLATE":    monitoringpb.Aggregation_ALIGN_INTERPOLATE,
+	"ALIGN_NEXT_OLDER":     monitoringpb.Aggregation_ALIGN_NEXT_OLDER,
+	"ALIGN_MIN":            monitoringpb.Aggregation_ALIGN_MIN,
+	"ALIGN_MAX":            monitoringpb.Aggregation_ALIGN_MAX,
+	"ALIGN_MEAN":           monitoringpb.Aggregation_ALIGN_MEAN,
+	"ALIGN_COUNT":          monitoringpb.Aggregation_ALIGN_COUNT,
+	"ALIGN_SUM":            monitoringpb.Aggregation_ALIGN_SUM,
+	"ALIGN_STDDEV":         monitoringpb.Aggregation_ALIGN_STDDEV,
+	"ALIGN_COUNT_TRUE":     monitoringpb.Aggregation_ALIGN_COUNT_TRUE,
+	"ALIGN_COUNT_FALSE":    monitoringpb.Aggregation_ALIGN_COUNT_FALSE,
+	"ALIGN_FRACTION_TRUE":  monitoringpb.Aggregation_ALIGN_FRACTION_TRUE,
+	"ALIGN_PERCENTILE_99":  monitoringpb.Aggregation_ALIGN_PERCENTILE_99,
+	"ALIGN_PERCENTILE_95":  monitoringpb.Aggregation_ALIGN_PERCENTILE_95,
+	"ALIGN_PERCENTILE_50":  monitoringpb.Aggregation_ALIGN_PERCENTILE_50,
+	"ALIGN_PERCENTILE_05":  monitoringpb.Aggregation_ALIGN_PERCENTILE_05,
+	"ALIGN_PERCENT_CHANGE": monitoringpb.Aggregation_ALIGN_PERCENT_CHANGE,
```

Comment:
I wonder if we should convert these to just `none`, `delta`, `rate`, `interpolate`, etc...While the current name you expose in the config is the one you can see in GCP docs, striping it may be more readable? I don't have a strong opinion here, especially having we will normally use the default aligner

Reply:
Hmmm I prefer keeping aligner consistent with GCP. When user uses the GCP API to query monitoring metrics, `ALIGN_NONE`, `ALIGN_MEAN` is the aligner format they have to use:
<img width="499" alt="Screen Shot 2020-04-22 at 8 44 30 AM" src="https://user-images.githubusercontent.com/14081635/79996285-85db4000-8475-11ea-99f7-e1dee0147339.png">